heli/assetlist.json

```diff
@@ -0,0 +1,34 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "helichain",
```

Comment:
This chain_name must match the directory name, so either chain_name has to change to 'heli' or the directory name must change to 'helichain'. This value is immutable.

Reply:
I updated chain_name